pkg/proxy/apis/config/types.go

```diff
@@ -22,6 +22,17 @@ import (
 	logsapi "k8s.io/component-base/logs/api/v1"
 )
 
+// IPVSIPSetConfiguration contains ipset-related configuration
+// details for IPVS proxier.
+type IPVSIPSetConfiguration struct {
```

Comment:
maybe `IPSetConfiguration`.
We don't use ipsets with iptables and nftables proxy, but theoretically it can be used and should be considered as independent entity which IPVS is consuming.

Reply:
or `KubeProxyIPSetConfiguration` which is consistent with pattern being followed here.

Reply:
I think you are right, `KubeProxyIPSetConfiguration` is better because the attributes of IPSet do not differ whether it is IPVS or IPTABLES, only the attribute values might differ.Therefore, I will change this name, but the configuration will still be within the IPVS structure.